libraries/search/lib/SearchSubject.ts

```diff
@@ -52,35 +52,21 @@ export abstract class SearchSubject<T extends Encoding> {
    * @param rootContext RootContext of the subject
    * @protected
    */
-  protected constructor(target: Target, rootContext: RootContext<unknown>) {
+  protected constructor(
+    target: Target,
+    rootContext: RootContext<unknown>,
```

Comment:
Do we still need the root context or can we also just pass the control flow program?

Reply:
Good catch, removed.